src/Compilers/VisualBasic/Portable/Analysis/FlowAnalysis/AbstractFlowPass.vb

```diff
@@ -339,14 +331,31 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Return False
             End If
             Select Case type.SpecialType
-                Case SpecialType.None,
-                     SpecialType.System_Nullable_T,
-                     SpecialType.System_IntPtr,
```

Comment:
>System_IntPtr [](http://example.com/codeflow?start=33&length=13)

I am not sure what was the motivation for excluding IntPtr/UIntPtr from the list of primitive types. More over, They are explicitly are considered as primitive  in ```IsPrimitiveType``` helper. Let's keep this logic, but I will open an issue to follow up.

Reply:
Opened #72316.